include/dca/phys/dca_step/cluster_solver/ctint/walker/tools/d_matrix_builder_gpu.hpp

```diff
@@ -28,36 +28,36 @@ namespace solver {
 namespace ctint {
 // dca::phys::solver::ctint::
 
-template <>
-class DMatrixBuilder<linalg::GPU> final : public DMatrixBuilder<linalg::CPU> {
+template <typename Real>
```

Comment:
So is this would be the place to document that in what way the GPU specialization of DMatrixBuilder is different from the CPU version.

Reply:
It only adds one method, I added a description there.